src/.config/nuxt.ts

```diff
@@ -113,6 +119,10 @@ export default defineNuxtConfig({
     },
     'nuxt-security',
   ],
+  shadcn: {
+    prefix: '',
```

Comment:
Should we add come prefix like `Shd` for shadcn?

(and maybe `Mvs` for maevsi...)